packages/protocol/contracts/common/FeeHandler.sol

```diff
@@ -681,7 +682,7 @@ contract FeeHandler is
       "Can't distribute to the zero address"
     );
     IERC20 token = IERC20(tokenAddress);
-    uint256 tokenBalance = token.balanceOf(address(this));
+    token.balanceOf(address(this));
```

Comment:
Can this line be removed?

Reply:
This is necessary to forge forge to compile import "@test-sol/unit/governance/validators/mocks/ValidatorsMock.sol";

Reply:
I believe the original idea was to test if tokenAddress implements IERC20

Reply:
On second though - removed